README.md

```diff
@@ -14,395 +14,245 @@
+anomalib train -h
+
+# To train the model on the default values, run:
+anomalib train --model anomalib.models.patchcore --data anomalib.data.MVTec
 ```
 
-where the currently available models are:
+</details>
 
-- [CFA](src/anomalib/models/cfa)
-- [CFlow](src/anomalib/models/cflow)
-- [DFKDE](src/anomalib/models/dfkde)
-- [DFM](src/anomalib/models/dfm)
-- [DRAEM](src/anomalib/models/draem)
-- [EfficientAd](src/anomalib/models/efficient_ad)
-- [FastFlow](src/anomalib/models/fastflow)
-- [GANomaly](src/anomalib/models/ganomaly)
-- [PADIM](src/anomalib/models/padim)
-- [PatchCore](src/anomalib/models/patchcore)
-- [Reverse Distillation](src/anomalib/models/reverse_distillation)
-- [STFPM](src/anomalib/models/stfpm)
+# 🤖 Inference
 
-## Feature extraction & (pre-trained) backbones
+Anomalib includes multiple inferencing scripts, including Torch, Lightning, Gradio, and OpenVINO inferencers to perform inference using the trained/exported model. Here we show an inference example using the Lightning inferencer. For other inferencers, please refer to the [Inference Documentation](https://anomalib.readthedocs.io).
 
-The pre-trained backbones come from [PyTorch Image Models (timm)](https://github.com/rwightman/pytorch-image-models), which are wrapped by `FeatureExtractor`.
+<details>
+<summary>Inference via API</summary>
+
+The following example demonstrates how to perform Lightning inference by loading a model from a checkpoint file.
+
+```python
+# Assuming the datamodule, model and engine is initialized from the previous step,
+# a prediction via a checkpoint file can be performed as follows:
+predictions = engine.predict(
+    datamodule=datamodule,
+    model=model,
+    ckpt_path="path/to/checkpoint.ckpt",
+)
+```
 
-For more information, please check our documentation or the [section about feature extraction in "Getting Started with PyTorch Image Models (timm): A Practitioner’s Guide"](https://towardsdatascience.com/getting-started-with-pytorch-image-models-timm-a-practitioners-guide-4e77b4bf9055#b83b:~:text=ready%20to%20train!-,Feature%20Extraction,-timm%20models%20also>).
+</details>
 
-Tips:
+<details>
+<summary>Inference via CLI</summary>
 
-- Papers With Code has an interface to easily browse models available in timm: [https://paperswithcode.com/lib/timm](https://paperswithcode.com/lib/timm)
+```bash
+# To get help about the arguments, run:
+anomalib predict -h
 
-- You can also find them with the function `timm.list_models("resnet*", pretrained=True)`
+# Predict via providing the model name
+anomalib predict --model anomalib.models.Patchcore
 
-The backbone can be set in the config file, two examples below.
+# Predict by overriding the various values with commands.
+anomalib predict --model anomalib.models.Patchcore --data <CONFIG | CLASS_PATH_OR_NAME>
 
-```yaml
-model:
-  name: cflow
-  backbone: wide_resnet50_2
-  pre_trained: true
+# Predict by using a config file.
+anomalib predict --config <config_file_path> --return_predictions
```

Comment:
If we give multiple examples here, maybe we should do the same for `anomalib train` above

Reply:
what would be your preference here? Remove the second example here, or add a second example to the other sections?